app/controllers/v0/institutions_controller.rb

```diff
@@ -94,7 +94,6 @@ def search_results
       filter_results(relation)
     end
 
-    # rubocop:disable Metrics/MethodLength
```

Comment:
Rubocop is your friend! Why not convert that gnarly array of arrays on line 99 into a frozen constant instead?